### ISSUE TITLE
Downloads from CI are using an invalid (internal) content hostname http://pulp-content-app:24816

### ISSUE THREAD

**Issue by ironfroggy** [ansible/galaxy_ng#185]
##### SUMMARY

Downloads from CI are using an invalid (internal) content hostname http://pulp-content-app:24816

Cannot download published collections from CI, as the content hostname appears to be misconfigured.

##### STEPS TO REPRODUCE

Select a published collection on https://ci.cloud.redhat.com/ansible/automation-hub/ and try to download it via `ansible-galaxy collection install ...`

##### EXPECTED RESULTS

To download the `.tar.gz` artifact and install it.

##### ACTUAL RESULTS

THe collection cannot be downloaded, and fails with a `ERROR! Unexpected Exception, this is probably a bug: <urlopen error [Errno -2] Name or service not known>`

```
ansible-galaxy 2.9.7
  config file = /tmp/tmpgoy3fzm3.cfg
  configured module search path = ['/home/calvin/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/calvin/projects/ansible/lib/ansible
  executable location = /home/calvin/.local/share/virtualenvs/iqe/bin/ansible-galaxy
  python version = 3.7.5 (default, Oct 17 2019, 12:21:00) [GCC 8.3.1 20190223 (Red Hat 8.3.1-2)]
Using /tmp/tmpgoy3fzm3.cfg as config file
Found installed collection orionuser1.collection_dep_a_tdfjfiqe:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/co
llection_dep_a_tdfjfiqe'
Found installed collection orionuser1.collection_dep_a_fpbgmitj:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/co
llection_dep_a_fpbgmitj'
Found installed collection orionuser1.collection_dep_a1_eukthupm:1.2.3-rc1 at '/home/calvin/.ansible/collections/ansible_collections/orionuse
r1/collection_dep_a1_eukthupm'
Found installed collection orionuser1.collection_dep_a_ajrfbhjk:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/co
llection_dep_a_ajrfbhjk'
Found installed collection orionuser1.collection_dep_a_lbzxvits:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/co
llection_dep_a_lbzxvits'
Found installed collection orionuser1.collection_dep_a1_vdgzxseg:2.1113.36 at '/home/calvin/.ansible/collections/ansible_collections/orionuse
r1/collection_dep_a1_vdgzxseg'
Found installed collection orionuser1.collection_dep_a_bxyivcvs:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/co
llection_dep_a_bxyivcvs'
Found installed collection orionuser1.collection_dep_a_jlctwrct:1.0.0 at '/home/calvin/.ansible/collections/ansible_collections/orionuser1/collection_dep_a_jlctwrct'
Process install dependency map
Processing requirement collection 'autohubtest2.collection_dep_a_tquemovm'
Collection 'autohubtest2.collection_dep_a_tquemovm' obtained from server automation_hub https://ci.cloud.redhat.com/api/automation-hub/
Starting collection install process
Installing 'autohubtest2.collection_dep_a_tquemovm:1.0.1' to '/home/calvin/.ansible/collections/ansible_collections/autohubtest2/collection_dep_a_tquemovm'
Downloading http://pulp-content-app:24816/api/automation-hub/v3/artifacts/collections/automation-hub/autohubtest2.collection_dep_a_tquemovm.1.0.1.tar.gz to /home/calvin/.ansible/tmp/ansible-local-2042irqci16y/tmpoxl1m7cu
ERROR! Unexpected Exception, this is probably a bug: <urlopen error [Errno -2] Name or service not known>
```

**Comment by alikins** [comment]
I can reproduce this.

`Downloading http://pulp-content-app:24816/api/automation-hub/v3/artifacts/collections/automation-hub/autohubtest2-collection_dep_a_obddfjkm-1.0.0.tar.gz to /home/adrian/.ansible/tmp/ansible-local-18052159s_1_j2e/tmpjhs_6y4s`

**Comment by alikins** [comment]
This is addressed in the e2e-deploy pr #1730  https://github.com/RedHatInsights/e2e-deploy/pull/1730

It was manually deployed to ci, and not htat #1730 is merged to e2e-deploy this shouldn't be an issue in the future.

**Comment by alikins** [comment]
@ironfroggy @cutwater Since https://github.com/RedHatInsights/e2e-deploy/pull/1730 is merged, this should be resolved now and going forward.
